airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -699,6 +699,76 @@ class FormStream(Stream):
     created_at_field = "createdAt"
 
 
+class PropertyHistoryStream(IncrementalStream):
+    """Contacts Endpoint, API v1
+    Is used to get all Contacts and the history of their respective
+    Properties. Whenever a property is changed it is added here.
+    Docs: https://legacydocs.hubspot.com/docs/methods/contacts/get_contacts
+    """
+    more_key = "has-more"
+    url = "/contacts/v1/lists/all/contacts/all"
+    updated_at_field = "timestamp"
+    created_at_field = "timestamp"
+    data_field = "contacts"
+    page_field = "vid-offset"
+    page_filter = "vidOffset"
+
+    def list(self, fields) -> Iterable:
+        properties = self._api.get(f"/properties/v2/contact/properties")
```

Comment:
I don’t understand why we can’t just define the `contact` for this stream as `self.entity` and not override the `list` method, it will be executed as in other streams, right?